libs/providers/flagd-web/src/e2e/setup.ts

```diff
@@ -11,11 +12,11 @@ OpenFeature.setProvider(
     host: 'localhost',
     port: 8013,
     tls: false,
-    maxRetries: -1,
```

Comment:
This is the cause of your `--force-exit` / open handler issues. Without this param, the provider tries to reconnect to flagd (this socket is the open handle). Add it to where you now create this provider and things work again :)

Reply:
oh thank you, this makes so much sense